lib/gems/pending/util/vmdb-logger.rb

```diff
@@ -156,8 +156,9 @@ class Formatter < Logger::Formatter
     FORMAT = "[----] %s, [%s#%d:%x] %5s -- %s: %s\n"
 
     def call(severity, time, progname, msg)
-      msg = prefix_task_id(msg2str(msg))
+      max_log_line_length = 1.megabyte # TODO: (julian) Move to config
```

Comment:
Don't worry about config, because I can't see a user configuring this.

Reply:
Ok thanks @Fryguy.

Reply:
You might need to add some activesupport extensions at the top of this file...it can run standalone without the rails env.  See how we use core_ext above.